scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala

```diff
@@ -152,52 +160,68 @@ object SymOps:
         else termParamss(1).params(0)
       }
```

Comment:
`extendedSymbol` should probably also be updated, as it implicitly uses the old heuristic